compiler/modules/err/err.bal

```diff
@@ -53,15 +53,23 @@ public function location(File file, Position? startPos = (), Position? endPos =
     };
 }
 
-public function syntax(Message m, Location? loc = (), string? functionName = (), error? cause = ()) returns Syntax {
+public function locationFromFilename(string filename, LineColumn? startPos = (), LineColumn? endPos = ()) returns Location {
```

Comment:
This is just for test cases - in tests/ somewhere, not part of the public API of bal. Plus for test cases you don't need startPos and endPos.

Reply:
fixed